src/common/selftest/selftest_firstlayer_interface.cpp

```diff
@@ -0,0 +1,52 @@
+/**
+ * @file selftest_firstlayer_interface.cpp
+ */
+#include "selftest_firstlayer_interface.hpp"
+#include "selftest_firstlayer.hpp"
+#include "selftest_sub_state.hpp"
+#include "selftest_part.hpp"
+#include "eeprom.h"
+
+namespace selftest {
+SelftestFirstLayer_t staticResult; // automatically initialized by PartHandler
+
+bool phaseFirstLayer(IPartHandler *&pFirstLayer, const FirstLayerConfig_t &config) {
```

Comment:
Missing documentation. Purpose of function, its parameters and return value are not documented.

Reply:
fixed